### DISCUSSION TITLE
ASA Stats community priority for the following months is to prepare some of our project's milestones for a grant

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#554] [Ideas]
https://github.com/algorandfoundation/xGov

ASA Stats community priority for the following months is to prepare some of our project's milestones for a grant, to create and conduct a campaign all over the ecosystem's social media channels.

Let's face it, whales who invested in projects will choose their project to get a grant, but for the sake of long-term xGov process sustainability from time to time they simply have to pick some independent project.

https://github.com/asastats/docs/blob/main/roadmap.md

In short, everybody says that everybody in the ecosystem uses ASA Stats, so let's see if that's true in the following months.

**Comment by ipaleka** [comment]
[ASA Stats Point in Time - tax reporting reference for US citizens](https://github.com/algorandfoundation/xGov/pull/17)
